app/qml/CodeScanner.qml

```diff
@@ -11,72 +11,74 @@ import QtQuick
 import QtQuick.Controls
 import QtQuick.Layouts
 import QtMultimedia
-import lc 1.0
+import lc 1.0 as InputClass
 
 import "./components"
 
-Drawer {
-  id: codeReader
-  palette.dark: InputStyle.fontColor // changes busy indicator color
+Item {
```

Comment:
Let's make the `Page` a top-level element. There should be no need to wrap it inside an `Item`